### ISSUE TITLE
Missing a visual completion state other than colour

### ISSUE THREAD

**Issue by kirsty-hames** [cgkineo/adapt-hotgrid#99]
Viewed/completed items indicate their completed state with a change in image or border colour. This information will be difficult or impossible to perceive for individuals with visual disabilities such as low vision or color blindness.

Provide a non-color dependent means of identifying viewed/completed items. For Hotgraphic, this was implemented with a checkmark icon over each pin to indicate the completed state - https://github.com/adaptlearning/adapt-contrib-hotgraphic/pull/231

**Comment by swashbuck** [comment]
Hi, @kirsty-hames . I was thinking of adding a **checkmark icon** sort of like this but with Vanilla styles / icons. Thoughts?

![icon](https://user-images.githubusercontent.com/898168/226474251-5e5db1a4-e59c-409a-8c96-0749b1d048e1.jpg)

**Comment by kirsty-hames** [comment]
> Hi, @kirsty-hames . I was thinking of adding a **checkmark icon** sort of like this but with Vanilla styles / icons. Could do something similar when the images are _not_ rounded, too. Thoughts?

Hey @swashbuck, I think the checkmark icon is a good solution as this makes it consistent with other components (both Hotgraphic and Accordion display a checkmark when visited).
